objects/o_solid_oneway/Create_0.gml

```diff
@@ -0,0 +1,114 @@
+#region Initialize
+	
+	//movement variables
+	xspd = 0;
+	xspd_remainder = 0;
+
+	yspd = 0;
+	yspd_remainder = 0;
+
+	dir = 1; //movement direction
+	
+#endregion
+
+#region Utils
+
+	//populate a list of all actors who are riding the solid
+	//Returns: a DS list of actors riding the solid
+	function get_rider_list() {
+		var _list_of_riders = ds_list_create();
+		ds_list_clear(_list_of_riders);
```

Comment:
We should make a script with this in it. Also, it would be good if we can find a way to re-use the list, I think originally it was, since you're calling `ds_list_clear here`. That function doesn't really make sense because it's created on the line above so it's already empty